### PR TITLE
Add User creation test

### PR DESCRIPTION
**Pull request by poflankov**

### CODE REVIEW

File: cypress/integration/users/create-users.spec.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    cy.login('diku_admin', 'admin');
+  });
+
+  it('should be possible', function () {
+    Cypress.on('uncaught:exception', () => {
+      return false;
+    });
+
+    cy.visit('/users/create');
+    cy.get('#adduser_lastname').type('Test123');
```
Comment (NikitaSedyx) [inline cypress/integration/users/create-users.spec.js:12]: please use interactors instead

additionally wrap it in commands, e.g. fillUserInfo - more reusable pieces of code

File: cypress/integration/users/create-users.spec.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+describe('Creating user', () => {
+  beforeEach(() => {
+    cy.login('diku_admin', 'admin');
+  });
+
+  it('should be possible', function () {
+    Cypress.on('uncaught:exception', () => {
```
Comment (NikitaSedyx) [inline cypress/integration/users/create-users.spec.js:7]: do you really need this one? additionally why is it defined in test case?

File: cypress/integration/users/create-users.spec.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      return false;
+    });
+
+    cy.visit('/users/create');
+    cy.get('#adduser_lastname').type('Test123');
+    cy.get('#adduser_group').select('undergrad (Undergraduate Student)', { force: true });
+    cy.get('#adduser_email').type('test@folio.org', { force: true });
+    cy.get('#clickable-save').click({ force: true });
```
Comment (NikitaSedyx) [inline cypress/integration/users/create-users.spec.js:15]: please follow AAA pattern

File: cypress/integration/users/create-users.spec.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    cy.get('#pane-userdetails:contains(Test123)');
+    cy.get('#pane-userdetails [data-pane-header-actions-dropdown="true"]').click();
```
Comment (NikitaSedyx) [inline cypress/integration/users/create-users.spec.js:17]: if you want to clean up data, use API for this, your test case is about user creation, you shouldn't click delete buttons in it

File: cypress/integration/users/create-users.spec.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+} from "../../../interactors";
+
+describe('Creating user', () => {
+  beforeEach(() => {
+    cy.login('diku_admin', 'admin');
+  });
+
+  it('should be possible', function () {
+    cy.visit('/users/create');
+    cy.createUser('Test123', 'undergrad (Undergraduate Student)', 'test@folio.org');
```
Comment (NikitaSedyx) [inline cypress/integration/users/create-users.spec.js:12]: `undergrad (Undergraduate Student)` name can't be constant - firstly fetch available patron groups via API and pass one of their names to form

File: cypress/integration/users/create-users.spec.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import {
+  Pane,
+} from "../../../interactors";
+
+describe('Creating user', () => {
+  beforeEach(() => {
+    cy.login('diku_admin', 'admin');
+  });
+
+  it('should be possible', function () {
```
Comment (NikitaSedyx) [inline cypress/integration/users/create-users.spec.js:10]: name doesn't say a lot

File: cypress/integration/users/create-users.spec.js (hunk 4 above)
Comment (NikitaSedyx) [reply cypress/integration/users/create-users.spec.js:15]: explained offline

File: cypress/support/users.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import { Button, Select, TextField } from "../../interactors";
+
+Cypress.Commands.add('createUser', (userLastName, patronGroup, email) => {
+  cy.do([
+    TextField('Last name\n*').fillIn(userLastName),
```
Comment (NikitaSedyx) [inline cypress/support/users.js:5]: id/attr is more preferable

Comment (cowboyd) [reply cypress/support/users.js:5]: You can also a matcher if you don't want to include the `\n*`:

```js
import { including } from "../../interactors"

// .....
cy.do([
  TextField(including('Last name')).fillIn(userLastName),
  //....
]);

File: cypress/integration/users/create-users.spec.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+  it('should be possible by filling the "Create user" form and submitting it', function () {
+    const lastName = 'Test123' + Number(new Date()).toString();
+    const userGroupOption = Cypress.env('userGroups')[0].group + ' (' + Cypress.env('userGroups')[0].desc + ')';
+
+    cy.visit('/users/create');
+    cy.createUser(lastName, userGroupOption, 'test@folio.org');
+    cy.expect(Pane(including(lastName)).is({ visible: true, index: 2 }));
+
+    cy.getUsers({ query: `personal.lastName="${lastName}"` })
```
Comment (NikitaSedyx) [inline cypress/integration/users/create-users.spec.js:24]: delete should be done in afterEach, data clean up is not realted to test case

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**